packages/dds/tree/src/core/tree/anchorSet.ts

```diff
@@ -131,6 +131,29 @@ export interface AnchorEvents {
 	 */
 	subtreeChanging(anchor: AnchorNode): PathVisitor | void;
 
+	/**
+	 * Emitted when exiting the subtree rooted at `anchor` during a delta visit, which means that all effects of that
+	 * visit in that subtree are now visible. HOWEVER, as of 2024-03-27, applying changes to a tree requires two visits
+	 * (the detach and the attach passes) so this event fires twice.
```

Comment:
Describing the contract for this event in terms of the delta visit behavior is doesn't seem ideal (the existence of delta, and the particulars of the visit algorithm for them, are implementation details that users of this even likely won't and shouldn't know about). Maybe something like this would be more approachable:

Emitted after the subtree rooted at `anchor` may have been changed.
While this event is always emitted in the presence of changes to the subtree,
it may also be emitted even though no changes have been made to the subtree.
May be emitted multiple times within the application of a single edit or transaction.